api/src/test/java/org/openmrs/api/ProgramWorkflowServiceUnitTest.java

```diff
@@ -67,4 +72,15 @@ public void getProgramByName_shouldFailWhenTwoProgramsFoundWithSameName() {
 		pws.setProgramWorkflowDAO(mockDao);
 		pws.getProgramByName("A name");
 	}
+	
+	@Test
+	public void saveProgramme_shouldThrowErrorIfConceptOfProgrammeIsNull() {
+		
+		thrown.expect(APIException.class);
+		thrown.expectMessage("Program concept is required");
+		
+		Program program1 = new Program( 1 );
```

Comment:
@dilekamadushan please dont write `new Program( 1 );` instead `new Program(1);` we do not put spaces around our parameters. Same below when calling `saveProgram` Fix your IDE style configuration if your IDE is doing that automatically https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE